group_vars/location_e16outdoor/networks.yml

```diff
@@ -53,6 +52,7 @@ networks:
     name: mesh_segen
     prefix: 10.31.142.35/32
     ipv6_subprefix: -4
+    mesh_metric: 128
```

Comment:
please leave 5GHz mesh_metrics at default (512). We want to retain some traffic engineering margin in order to prefer faster links.

Reply:
changed. will go up on next firmware update.